Source/WebInspectorUI/UserInterface/Views/OverrideDeviceSettingsPopover.js

```diff
@@ -0,0 +1,324 @@
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+ * THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+WI.OverrideDeviceSettingsPopover = class OverrideDeviceSettingsPopover extends WI.SettingsPopover
+{
+    constructor(delegate)
+    {
+        super(delegate);
+    }
+
+    // Protected
+
+    createContentElement()
+    {
+        let contentElement = document.createElement("div");
+        contentElement.classList.add("device-settings-content");
+
+        if (InspectorFrontendHost.isRemote)
+            contentElement.appendChild(this._createUserAgentSection());
+
+        if (WI.DeviceSettingsManager.supportsSetScreenSizeOverride())
+            contentElement.appendChild(this._createScreenSizeSection());
+
+        contentElement.appendChild(this._createSettingsSection());
+        contentElement.appendChild(WI.ReferencePage.DeviceSettings.createLinkElement());
+
+        return contentElement;
+    }
+
+    // Private
+
+    _createSettingsSection()
+    {
+        let coreSettingsGroup = {
+            name: WI.UIString("Core Features"),
+            settings: [
+                {name: WI.UIString("Disable Images"), setting: InspectorBackend.Enum.Page.Setting.ImagesEnabled, value: false},
+                {name: WI.UIString("Disable CSS"), setting: InspectorBackend.Enum.Page.Setting.AuthorAndUserStylesEnabled, value: false},
+                {name: WI.UIString("Disable JavaScript"), setting: InspectorBackend.Enum.Page.Setting.ScriptEnabled, value: false},
+            ],
+        };
+
+        let compatibilitySettingsGroup = {
+            name: WI.UIString("Compatibility"),
+            settings: [
+                {name: WI.UIString("Disable site-specific hacks"), setting: InspectorBackend.Enum.Page.Setting.NeedsSiteSpecificQuirks, value: false},
+            ],
+        };
+
+        let securitySettingsGroup = {
+            name: WI.UIString("Security"),
+            settings: [
+                {name: WI.UIString("Disable cross-origin restrictions"), setting: InspectorBackend.Enum.Page.Setting.WebSecurityEnabled, value: false},
+            ],
+        };
+
+        let privacySettingsGroup = {
+            name: WI.UIString("Privacy"),
+            settings: [
+                {name: WI.UIString("Enable Intelligent Tracking Prevention debug mode"), setting: InspectorBackend.Enum.Page.Setting.ITPDebugModeEnabled, value: true},
+                // COMPATIBILITY (iOS 14.0): `Page.Setting.AdClickAttributionDebugModeEnabled` was renamed to `Page.Setting.PrivateClickMeasurementDebugModeEnabled`.
+                {name: WI.UIString("Enable Private Click Measurement debug mode"), setting: InspectorBackend.Enum.Page.Setting.PrivateClickMeasurementDebugModeEnabled, value: true},
+
+                // FIXME: Find out whether we need to keep this.
+                // {name: WI.UIString("Ad Click Attribution Debug Mode"), setting: InspectorBackend.Enum.Page.Setting.AdClickAttributionDebugModeEnabled, value: true},
```

Comment:
why wouldn't we still want this?  even if this is no longer used (and i think it still is?) we'd still wanna show it for when inspecting older platforms

Reply:
Yeah, we will want to keep this for inspecting older platform. To match the intended capitalization it should be "Ad Click Attribution debug mode"

Reply:
Got it. I had misunderstood the setting until I re-read the compatibility comment. 
Refactored to add the the settings according to support:

```javascript
// COMPATIBILITY (iOS 14.0): `Page.Setting.AdClickAttributionDebugModeEnabled` was renamed to `Page.Setting.PrivateClickMeasurementDebugModeEnabled`.
if (InspectorBackend.Enum.Page.Setting.PrivateClickMeasurementDebugModeEnabled)
    privacySettingsGroup.settings.push({name: WI.UIString("Enable Private Click Measurement debug mode"), setting: InspectorBackend.Enum.Page.Setting.PrivateClickMeasurementDebugModeEnabled, value: true});

// COMPATIBILITY (iOS 14.0): `Page.Setting.AdClickAttributionDebugModeEnabled` was renamed to `Page.Setting.PrivateClickMeasurementDebugModeEnabled`.
if (InspectorBackend.Enum.Page.Setting.AdClickAttributionDebugModeEnabled)
    privacySettingsGroup.settings.push({name: WI.UIString("Ad Click Attribution debug mode"), setting: InspectorBackend.Enum.Page.Setting.AdClickAttributionDebugModeEnabled, value: true});
```